doc/PACKAGING

```diff
@@ -0,0 +1,198 @@
+EARLY VERSION
+EARLY VERSION
+The "mod" section is not written yet! 
+EARLY VERSION
+EARLY VERSION
+
+This document describes how packages are done in unvanquished.
+
+Games running in Daemon-engine are only able to see files
+inside a VFS (Virtual File System), built mostly from
+packages contents (and some other files server-side).
+
+Packages are compressed archives, which can be generated by
+standard archivers.
+
+To create such an archive, place all and only the needed
+files (beware hiden files, they can prevent the game to load
+packages!) in a folder, and then run.
```

Comment:
It might be useful to mention conceptually what you are doing is creating a zip file and renaming it to have a .dpk extension.